### ISSUE TITLE
Being able to pour Nitroglycerine and it Can be Lit on Fire

### ISSUE THREAD

**Issue by fatoastcatbruh** [studio-minus/ppg-bugs-and-requests#2345]
**Feature description**
you would be able to pour nitro on houses people etc. and then light it and watch it burn

**Feature purpose**
destruction

**Additional context**
idk i wanna burn things

**Comment by Drogo-man** [comment]
This isnt related to any of these lmao

**Comment by DindinYT37** [comment]
> This isnt related to any of these lmao

"Water is not pouring" is a good example already, and that's just considering the titles.

**Comment by Dawid23-Gamer** [comment]
> Duplicate of or Related to:
> 
> * [Showerhead water interaction #359](https://github.com/studio-minus/ppg-bugs-and-requests/issues/359)
> * [Spawnable water? #597](https://github.com/studio-minus/ppg-bugs-and-requests/issues/597)
> * [Water #792](https://github.com/studio-minus/ppg-bugs-and-requests/issues/792)
> * [Water is not pouring #849](https://github.com/studio-minus/ppg-bugs-and-requests/issues/849)
> * [Puddles #941](https://github.com/studio-minus/ppg-bugs-and-requests/issues/941)
> * [Spawnable liquids + Weather #1234](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1234)
> * [Realistic Water #1325](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1325)
> * [water releasing power #1791](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1791)

Uh, isn't this post about an in-game liquid changing the property of an item when it's poured on it, which is already a feature in the game (with a pouring liquid entering empty containers)? In what way does water physics relate to this?

**Comment by DindinYT37** [comment]
> Uh, isn't this post about an in-game liquid changing the property of an item when it's poured on it, which is already a feature in the game (with a pouring liquid entering empty containers)? In what way does water physics relate to this?

Yeah, but I thought it could imply that you would also be able to pour it on the ground as an example – which would mean he was indirectly asking for liquid physics.

**Comment by BrianTheVoid** [comment]
use the PPG Additions mod.
